docs/clients/dotnet/20.10/connecting/compatibility-mode.md

```diff
@@ -0,0 +1,18 @@
+# Compatibility Mode
+
+Compatibility Mode was added in v5.0.10 and v21.2.0 of the client to set certain configuration options depending on the Event Store version the client is connecting to. It allowa discovering whether the Event Store server is a v5 or v20 server.
```

Comment:
```suggestion
Compatibility Mode was added in v5.0.10 and v21.2.0 of the client to set certain configuration options depending on the Event Store version the client is connecting to. It allows discovering whether the Event Store server is a v5 or v20 server.
```

Reply:
Fixed in https://github.com/EventStore/documentation/pull/288/commits/151543c1acbb559165d3d983279156be408f5eb2.